origami.json

```
@@ -1,5 +1,6 @@
 {
-	"description": "Responsive page header for minimally branded sites, eg B2B tools, documentation sites, and internal tools.",
+	"description": "Responsive page header for minimally branded sites, namely internal products, tooling, documentation sites, and customer products.",
```

Now o-header-services is customisable it may be used without minimal branding.
```suggestion
	"description": "Responsive page header for sites with minimal or customised branding, including internal products, customer facing tools, and specialist titles.",
```